gematria/granite/python/token_graph_builder_model.py

```diff
@@ -340,3 +358,76 @@ def _create_graph_network_modules(
             residual_connection=options.EnableFeature.BY_FLAG,
         ),
     )
+
+  def _make_batch_feed_dict(self) -> model_base.FeedDict:
+    feed_dict = super()._make_batch_feed_dict()
+    to_feed_instruction_annotations = None
+    if len(self._batch_graph_builder.instruction_annotations) == 0:
+      to_feed_instruction_annotations = np.zeros((0, 0), dtype=self.dtype)
+    else:
+      to_feed_instruction_annotations = np.array(
+          list(self._batch_graph_builder.instruction_annotations.values()),
```

Comment:
It would be nice if we could just `reshape()` the data provided from the graph builder, to avoid too much Python processing.